__tests__/app.test.js

```diff
@@ -12,6 +14,21 @@ afterAll(() => {
     return db.end();
 });
 
+describe("/api", () => {
+    test("GET - 200: Responds with an array of endpoint objects", () => {
+        return request(app)
+            .get("/api")
+            .expect(200)
+            .then((res) => {
+                const file = path.join(`${__dirname}/..`, "endpoints.json");
```

Comment:
How come you're using an npm package when you can use a normal file path or require the file in?